packages/core/src/plugins/startUrl.ts

```diff
@@ -102,8 +108,13 @@ export function pluginStartUrl(): DefaultRsbuildPlugin {
         const urls: string[] = [];
 
         if (startUrl === true || !startUrl) {
+          const { entry } = api.context;
+          const names = getRoutesName(entry);
           const protocol = https ? 'https' : 'http';
-          urls.push(`${protocol}://localhost:${port}`);
+          // auto open first entry
+          urls.push(
+            normalizeUrl(`${protocol}://localhost:${port}/${names[0]}`),
```

Comment:
If one of the names is `index`, can we just open `http://localhost:8080` instead of `http://localhost:8080/index` or `http://localhost:8080/xxx` ?

Reply:
done